scripts/copyrightsScript.js

```diff
@@ -0,0 +1,40 @@
+/*
+ Copyright (C) 2021 Shruti Agarwal (mail2shruti.ag@gmail.com), Gaurav Mishra (mishra.gaurav@siemens.com)
+
+ SPDX-License-Identifier: GPL-2.0
+
+ This program is free software; you can redistribute it and/or
+ modify it under the terms of the GNU General Public License
+ version 2 as published by the Free Software Foundation.
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License along
+ with this program; if not, write to the Free Software Foundation, Inc.,
+ 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+*/
+
+var csv = require("csv-parser");
+var fs = require("fs");
+
+var copyrights = [];
+
+fs.createReadStream("scripts/copyrights.csv")
+  .pipe(csv())
+  .on("data", function (data) {
+    copyrights.push(data.copyright);
+  })
+  .on("end", function () {
+    let uniqueCopyrights = [...new Set(copyrights)].sort();
+    fs.writeFile(
+      "scripts/copyrights.js",
+      JSON.stringify(uniqueCopyrights),
```

Comment:
Can the prettyfication of JSON be done here instead? Otherwise the linter will fail for each JSON update.

Reply:
@GMishx updated

Reply:
How about using `JSON.stringify(uniqueCopyrights, null, 2)` instead of `yarn format`?

Reply:
Yeah we can do that, but there is one concern of mine in it, we are hardcoding 2 right here if in future we changed the formating of our app then we have to change the scripts as well, rest as you suggest

Reply:
That should be fine as the styling guide changes every rarely.
Otherwise if `yarn format` can run specifically on the JSON then this approach is also good.

Reply:
@GMishx updated for the formatting for a paticular file.